src/wallet/wallettool.cpp

```diff
@@ -103,6 +104,11 @@ static void WalletShowInfo(CWallet* wallet_instance)
     tfm::format(std::cout, "Address Book: %zu\n", wallet_instance->m_address_book.size());
 }
 
+static bool SalvageWallet(const fs::path path)
```

Comment:
nit: could pass this through by reference

Reply:
Done